pkg/kubelet/apis/kubeletconfig/v1beta1/types.go

```diff
@@ -453,6 +453,10 @@ type KubeletConfiguration struct {
 	// Default: true
 	// +optional
 	CPUCFSQuota *bool `json:"cpuCFSQuota,omitempty"`
+	// cpuCFSQuotaPeriod is the CPU CFS quota period value, cpu.cfs_period_us
+	// Default: "100ms"
+	// +optional
+	CPUCFSQuotaPeriod *metav1.Duration `json:"cpuCFSQuotaPeriod,omitempty"`
```

Comment:
can you add a note about the impact of changing this field when using Dynamic Kubelet Config?

Reply:
done